libs/sdk-backend-bear/src/backend/workspace/measures/index.ts

```diff
@@ -154,4 +157,39 @@ export class BearWorkspaceMeasures implements IWorkspaceMeasuresService {
 
         return measure;
     }
+
+    async getMeasureReferencingObjects(ref: ObjRef): Promise<IMeasureReferencing> {
+        const uri = await objRefToUri(ref, this.workspace, this.authCall);
+        const objectId = getObjectIdFromUri(uri);
+
+        const measures = await this.authCall(async (sdk) => {
+            const usedBy = await sdk.xhr.getParsed<{ entries: GdcMetadata.IObjectXrefEntry[] }>(
+                `/gdc/md/${this.workspace}/usedby2/${objectId}?types=metric`,
```

Comment:
Is it possible to use `sdk.md.getObjectUsedBy` method in bear client, or is there something missing/different? Also, you should be able to query visualizationObject & metric metadata objects at once like this. (by providing both types in options.types param). If the method is not suitable, please parallelise these 2 requests by Promise.all() (they don't depend on each other)